lib/libc/newlib/CMakeLists.txt

```diff
@@ -7,8 +7,12 @@ zephyr_library_sources(libc-hooks.c)
 # unable to, it will be up to the user to specify LIBC_*_DIR vars to
```

Comment:
> The one exception is the path to the newlib headers.

The one exception to what?

Reply:
don't follow

Reply:
> When we build with newlib we don't set -nostdinc.  In that case make
> sure that we leave it to the toolchain to set the system include paths.
> 
> The one exception is the path to the newlib headers.  

I'm not able to tell which rule "The one exception" is referring to.

Reply:
Ah, now that the previous sentence is fixed, I can tell.

Reply:
Updated the commit message, see if that's more clear.

Reply:
LGTM